assets/differentialabundance_report.Rmd

```diff
@@ -232,22 +233,73 @@ assay_names <- simpleSplit(params$exploratory_assay_names)
 names(assay_names) = assay_names
 assay_files <- lapply(assay_names, function(x) params[[paste0(x, '_matrix')]])
 
-assay_data <- lapply(assay_files, function(x) {
-  mat <- read_matrix(
-    x,
-    sample_metadata = observations,
-    row.names = 1
+# Set up vector of unlogged assay files (if any)
+unlogged <- c()
+features_log2_assays <- params$features_log2_assays
+
+# For maxquant input override param as all assays are already logged by proteus
+if (params$study_type == 'maxquant') {
+  features_log2_assays <- ""
+}
+
+if (is.null(features_log2_assays)) {
```

Comment:
@WackerO you resolved this, but you're still repeating code here rather than using the factored-out functionality. To be clear, I meant [this function](https://github.com/pinin4fjords/shinyngs/blob/b544d858e0c64879b12b5542c3e7cd0141d939a0/R/accessory.R#L1291). You can it being used in the exploratory script [here](https://github.com/pinin4fjords/shinyngs/blob/b544d858e0c64879b12b5542c3e7cd0141d939a0/exec/exploratory_plots.R#L189).

Reply:
Ah of course, just had to update shinyngs for the rmarkdown report for that function to be found. I have for now added the [] removal in the report; maybe in a future release that part can be integrated into shiny as well